src/components/DelimitedList/DelimitedList.stories.js

```diff
@@ -3,46 +3,33 @@
  * @copyright IBM Security 2019
  */
 
-import { withA11y } from '@storybook/addon-a11y';
-import centered from '@storybook/addon-centered/react';
+import { breakpoints } from '@carbon/layout';
+
 import { array, boolean, text } from '@storybook/addon-knobs';
 import { storiesOf } from '@storybook/react';
 
 import React from 'react';
 
-import { settings } from 'carbon-components';
-import { components } from '../../../.storybook';
+import { components, info } from '../../../.storybook';
 
 import { DelimitedList } from '../..';
 
-import props from './_mocks_';
-
-const { prefix } = settings;
+import mocks from './_mocks_';
 
 const { delimiter, truncate } = DelimitedList.defaultProps;
 
-const storyProps = () => ({
-  delimiter: text('Delimiter (`delimiter`)', delimiter),
-  items: array('Items (`items`)', props.items),
-  truncate: boolean('Truncate (`truncate`)', truncate),
+const props = () => ({
+  items: array('Items (items)', mocks.items),
+  truncate: boolean('Truncate (truncate)', truncate),
+  delimiter: text('Delimiter (delimiter)', delimiter),
+  style: { width: breakpoints.sm.width },
```

Comment:
I like this approach. 👍  While part of me still wishes we could include grid stuff here, I think this is probably for the best. 🤔 It keeps the story source from getting clogged up & makes everything simple + clean.